Components/Alerts/alerts.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="../../main.css">
+    <title>
+        Alerts
+    </title>
+</head>
+
+<body>
+    <!-- Navigation Top Bar Starts Here-->
+    <nav class="nav-container">
+        <div class="nav-bar-left">
+            <ul class="nav-bar-list">
+                <li class="nav-bar-title-text"><a class="link-a link-a-font-color-black" href="#">Sway UI</a></li>
+            </ul>
+        </div>
+        <div class="nav-bar-right">
+            <ul class="nav-bar-list">
+                <li class="nav-bar-list-items nav-bar-list-items-text">
+                    <a class="link-a link-a-font-color-black" href="#">Home</a>
+                </li>
+                <li class="nav-bar-list-items nav-bar-list-items-text">
+                    <a class="link-a link-a-font-color-black" href="#">Docs</a>
+                </li>
+                <li class="nav-bar-list-items nav-bar-list-items-text">
+                    <a class="link-a link-a-font-color-black" href="#">Github</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <!--Navbar Ends here-->
+
+    <div class="main-body-container">
+        <aside class="navbar-side">
+            <nav>
+                <ul class="sidebar-list">
+                    <li class="sidebar-list-items">
+                        <a href="../../Documentation/documentation.html" class="link-a link-a-font-color-black sidebar-list-items-text">Getting
+                            Started</a>
+                    </li>
+
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/Alerts/alerts.html" class="link-a link-a-font-color-black sidebar-list-items-text">Alerts</a>
+                    </li>
+                    <li class="sidebar-list-items">
+                        <a href="../../Components/" class="link-a link-a-font-color-black sidebar-list-items-text">Avatars</a>
```

Comment:
the naming is very big "link-a-font-color-black" .. stick to link-font or link-font-color like this for example